katello.dot

```diff
@@ -27,27 +27,27 @@ node "server" {
     [Foreman Discovery] as foreman_discovery
     [Foreman SCAP] as foreman_scap
 
-    foreman_tasks -down-> dynflow
-    katello -> foreman
-    katello -> foreman_tasks
-    katello -> foreman_docker
+    foreman_tasks -down..> dynflow
+    katello ..> foreman
+    katello ..> foreman_tasks
+    katello ..> foreman_docker
```

Comment:
for the plugins, you might use some non-directional arrow to indicate its 'connection' to foreman, as its not actually communication